Source/Scene/Scene.js

```diff
@@ -3563,5 +3653,15 @@ define([
         return SceneTransforms.wgs84ToWindowCoordinates(this, position, result);
     };
 
+    /**
+     * Requests a new rendered frame when {@link requestRenderMode} is set to <code>true</code>.
+     * The render rate will not exceed the {@link CesiumWidget#targetFrameRate}.
+     *
+     * @see Scene#requestRenderMode
+     */
+    Scene.prototype.requestRender = function() {
+        this._renderRequested = true;
+    };
```

Comment:
For organizational purposes this and `cartesianToCanvasCoordinates` should come before `isDestroyed`, and preferably closer to similar areas of the code.